staging/src/k8s.io/apiserver/pkg/storage/feature/feature_support_checker_test.go

```diff
@@ -260,8 +231,11 @@ func TestSupportsRequestWatchProgress(t *testing.T) {
 				mockClient := &MockEtcdClient{EndpointVersion: round.endpointsVersion}
 				ctx := context.Background()
 
-				err := testFeatureSupportChecker.CheckClient(ctx, mockClient, storage.RequestWatchProgress)
-				assert.Equal(t, err, round.expectedError)
+				// Function run in async mode error only applies for wrong feature that tested in TestSupports.
+				_ = testFeatureSupportChecker.CheckClient(ctx, mockClient, storage.RequestWatchProgress)
+
+				// Minimum wait time is 10 ms for first iteration, we expect it to run in 20 ms.
+				time.Sleep(20 * time.Millisecond)
```

Comment:
First time using `ExponentialBackoff` i couldn't find a better solution than this, please let me know if you have any.